packages/design-system-docs/src/scripts/components/CodeSnippet.jsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 
 const CodeSnippet = function (props) {
   return (
+    /* eslint-disable react/no-danger */
```

Comment:
I really like the `eslint-disable-next-line` that you used, can you use that here too?